roles/network_plugin/calico/defaults/main.yml

```
@@ -51,6 +51,10 @@ calico_bpf_log_level: ""
 # Valid option for service mode: Tunnel (default), DSR=Direct Server Return
 calico_bpf_service_mode: Tunnel
 
+# Calico floatingIPs support
+# Valid option for floatingIPs: Disabled (default), Enabled
+calico_felix_floatingIPs: Disabled
```

It is not good to mix camel cases and underscore cases like this.
For consistency, it is better to use
```suggestion
calico_felix_floating_ips: Disabled
```

---

Yes, indeed i'll change that. Thanks